scripts/jenkins/ardana/manual/lib.sh

```diff
@@ -111,6 +111,11 @@ function is_physical_deploy {
   [[ $ardana_env == qe* ]] || [[ $ardana_env == pcloud* ]]
 }
 
+function is_crowbar {
```

Comment:
You should use the `cloud_product` parameter to differentiate between the two products. Besides, using only the `crowbar` `scenario_name` for Crowbar may change soon, when we get the ability to generate crowbar batch files from any scenario.

Reply:
true, reworking